src/main/java/com/jcabi/dynamo/mock/MkTable.java

```diff
@@ -100,4 +100,17 @@ public Region region() {
     public String name() {
         return this.self;
     }
+
+    /**
+     * Delete item from aws table.
+     * @todo #8 Added delete method to Table interface. Only AwsTable method
```

Comment:
Please, be more precise in the task description. Imagine, you are the developer, and you get the task with the title "Added delete method to Table interface." What is that? What should I do, - you would think.
Good task description should start from "Implement MkTable.delete() operation ..."
This would be clear enough